tools/validators/instance_validator/tests/entity_instance_test.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for entity_instance.py"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from validate import generate_universe
+from validate import entity_instance
+from absl.testing import absltest
+
+import instance_parser
+import os
+
+_TESTCASE_PATH = os.path.join('.', 'tests', 'fake_instances')
+
+class EntityInstanceTest(absltest.TestCase):
```

Comment:
add test when the type id is missing?

Reply:
Because now I'm reading in the test YAML files with the syntax validator, the missing `id` case is handled by the YAML Schema